src/mrinufft/trajectories/inits/random_walk.py

```diff
@@ -20,8 +22,14 @@ def _get_neighbors_offsets(shape):
 
 
 def _initialize_ND_random_walk(
-    Nc, Ns, density, *, diagonals=True, pseudo_random=True, **sampling_kwargs
-):
+    Nc: int,
+    Ns: int,
+    density: np.typing.NDArray,
+    *,
+    diagonals: bool = True,
+    pseudo_random: bool = True,
+    **sampling_kwargs: Literal | bool,
```

Comment:
`Literal` has to be used as  `Literal["A string"]` or `Literal[1]` for instance

Reply:
Done. Most `Literal` uses were actually unjustified because of the way we handle `Enum` for everything